api/converter_utils.go

```diff
@@ -298,6 +308,94 @@ func txnRowToTransaction(row idb.TxnRow) (generated.Transaction, error) {
 	return txn, nil
 }
 
+func rowToBlock(blockHeader *sdk.BlockHeader) generated.Block {
```

Comment:
I haven't written much indexer code, but this name seems a bit underspecified. Above, we see `txnRowToTransaction`, so this should probably be `hdrRowToBlock`. I'm not totally confident about this recommendation because I would have expected this to take some sort of `idb` type, rather than an `sdk` type.

Reply:
That's fair, the name `hdrRowToBlock` will generate less confusion when reading the code.

Addressed in https://github.com/algorand/indexer/pull/1638/commits/eaefec7e83a256b20b77ab63ac75fd63e5874fca

I could create a specific `idb` type for the input parameter, but it would be a struct with only one field. Seemed a bit like overengineering or unnecessary abstraction. If the struct would have had more than one field, I believe it would have made sense.

It could go the other way though - "similar functions having similar signatures" (as in `hdrRowToblock` looking similar to `txnRowToTransaction`) is a desirable property to preserve.

Reply:
That's fair, the name `hdrRowToBlock` will generate less confusion when reading the code.

Addressed in https://github.com/algorand/indexer/pull/1638/commits/eaefec7e83a256b20b77ab63ac75fd63e5874fca and https://github.com/algorand/indexer/pull/1638/commits/8924c97009a56066562086b499273a822bbd4261